sim/Makefile

```diff
@@ -12,17 +12,15 @@ DUT = tb_didactic
 
 TB_DEFINES = \
 	+define+USE_UART \
+	+define+INC_ASSERT
 
 DUT_LIBS = \
-	-L $(BUILD_DIR)/pulp_lib \
-	-L $(BUILD_DIR)/ibex_lib \
-	-L $(BUILD_DIR)/riscv_dbg_lib \
-	-L $(BUILD_DIR)/tech_generic_lib \
 	-L $(BUILD_DIR)/didactic_lib \
 	-L $(BUILD_DIR)/tb_lib \
 
 VSIM_OPTS = \
 	-suppress vsim-3009 \
+	-suppress vsim-8386 \
```

Comment:
note: need to explain suppresses, both this new and the old ones as well (todo for myself)

Reply:
This suppresses `Error (suppressible): (vsim-8386) Illegal assignment to type 'enum reg[1:0] cdc_reset_ctrlr_pkg::clear_seq_phase_e' from type 'bit[1:0]': An enum variable may only be assigned the same enum typed variable or one of its values.`

So, assignment between type `reg` and type `bit`. Will add a comment to the Makefile.